src/collection/dimensions.js

```diff
@@ -925,6 +951,21 @@ defineDimFns( {
   name: 'height'
 } );
 
+elesfn.padding = function(){
+  if(this.length > 0) {
+    var ele = this[0];
+    var _p = ele._private;
+    if( _p.autoPadding !== undefined ) {
```

Comment:
Sorry I misunderstood your earlier change request. I have made changes so that it uses autoPadding if it is parent. Autopadding then only returns the pvalue if it is in pixel units, so a check might not be necessary.